src/main/java/frc/robot/subsystems/AimingSubsystem.java

```diff
@@ -0,0 +1,283 @@
+import frc.robot.constants.AimingConstants.AimState;
+
+
+public class AimingSubsystem extends SubsystemBase {
+  public enum AimingMotorMode {COAST, BRAKE}
+
+  // Elevator Hardware
+  private final TalonFX mLeftElevatorMotor = new TalonFX(AimingConstants.LEFT_ELEVATOR_TALON_ID,"rio");
+  private final TalonFX mRightElevatorMotor = new TalonFX(AimingConstants.RIGHT_ELEVATOR_TALON_ID,"rio");
+  
+  // TOF
+  private final TimeOfFlight mElevatorTOF = new TimeOfFlight(AimingConstants.ELEVATOR_TOF_ID);
+  
+  // Wrist Hardware
+  private final CANSparkMax mLeftWristMotor = new CANSparkMax(AimingConstants.LEFT_WRIST_SPARK_ID, MotorType.kBrushless);
+  private final CANSparkMax mRightWristMotor = new CANSparkMax(AimingConstants.RIGHT_WRIST_SPARK_ID, MotorType.kBrushless);
+  private final DutyCycleEncoder mWristThroughBoreEncoder = new DutyCycleEncoder(AimingConstants.ELEVATOR_THROUGHBORE_ENCODER_ID);
+
+  // Current States
+  private double mCurrentElevatorDistanceIn = AimingConstants.MIN_ELEVATOR_DIST_IN;
+  private double mCurrentWristRotationDeg = AimingConstants.MIN_WRIST_ROTATION_DEG;
+
+  // Desired States
+  private double mDesiredElevatorDistanceIn = mCurrentElevatorDistanceIn;
+  private double mDesiredWristRotationDeg = mCurrentWristRotationDeg;
+
+  // MotorMode Chooser
+  private final SendableChooser<AimingMotorMode> mChooser = new SendableChooser<>();
+
+  // PID Controllers and Motor Configs 
+  PIDController mElevatorController = AimingConstants.mElevatorPIDConstants.toWPIController();  
+  PIDController mWristController = AimingConstants.mWristPIDConstants.toWPIController();
+
+  ArmFeedforward mWristFeedforwardController = new ArmFeedforward(AimingConstants.mWristPIDConstants.mKS, AimingConstants.mWristkG, AimingConstants.mWristPIDConstants.mKS);
+  ElevatorFeedforward mElevatorFeedforward = new ElevatorFeedforward(AimingConstants.mElevatorPIDConstants.mKS, AimingConstants.mElevatorkG, AimingConstants.mElevatorPIDConstants.mKV);
+
+  MotorOutputConfigs mLeftWristMotorOutputConfigs = new MotorOutputConfigs();
+  MotorOutputConfigs mRightWristMotorOutputConfigs = new MotorOutputConfigs();
+
+  Slot0Configs mElevatorControllerSlot0Configs = new Slot0Configs();
+
+  // Onboard PID
+  // PositionVoltage elevatorVoltage = new PositionVoltage(getDesiredElevatorDistance()).withSlot(0);
+
+  public AimingSubsystem() {
+    mChooser.addOption("Brake", AimingMotorMode.BRAKE);
+    mChooser.addOption("Coast", AimingMotorMode.COAST);
+    mChooser.setDefaultOption("Brake", AimingConstants.INITIAL_MOTOR_MODE);
+
+    //Initializes TOF Sensor
+    setTOFViewZone(8, 8, 12, 12);
+    setTOFRange("short", 20);
+    configureDevices();
+  }
+
+  // Setting Conversions and Inversions
+  public void configureDevices() {
+
+    //initialize PID Controller Constants
+    mElevatorController.setPID(AimingConstants.mElevatorPIDConstants.mKP, AimingConstants.mElevatorPIDConstants.mKI, AimingConstants.mElevatorPIDConstants.mKD);
```

Comment:
This is redundant since you've already set it using the PIDParameters

Reply:
Fixed now.